src/NuGet.DependencyResolver.Core/Providers/IDependencyProvider.cs

```diff
@@ -12,5 +12,7 @@ public interface IDependencyProvider
         bool SupportsType(string libraryType);
 
         Library GetDescription(LibraryRange libraryRange, NuGetFramework targetFramework);
+
+        IEnumerable<string> GetAttemptedPaths(NuGetFramework targetFramework);
```

Comment:
Bringing this back because Lou's `runtime.json` feature needs it: https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Framework.PackageManager/Restore/LocalWalkProvider.cs#L54